app/services/ApiService.ts

```diff
@@ -89,24 +89,24 @@ export const deleteApiKey = (): Promise<void> => {
 };
 
 // --- KYC --- //
-export const putKycData = (data: KycData): Promise<void> => {
-  return fetchFrom(`${KycUrl}/data`, "POST", toKycDataDto(data));
+export const putKycData = (data: KycData, code?: string): Promise<KycInfo> => {
```

Comment:
Why is the code an optional param? (Just realising it now)
If it isn't initialised the fetch is …/kyc/**undefined**/data

This **ALWAYS** fails if code is not defined!

-->
```
  "message": "Error: Conversion failed when converting from a character string to uniqueidentifier.",
  "statusCode": 500,
```